services/brig/brig.integration.yaml

```diff
@@ -73,7 +73,8 @@ zauth:
 
 turn:
   # Replace this with the list of supported URIs
-  servers: test/resources/turn/servers.txt
+  serversV1: test/resources/turn/servers-v1.txt
+  serversV2: test/resources/turn/servers-v2.txt
```

Comment:
Let's use `servers` and `serversV2` (consistent with endpoint naming) to avoid having to lock the configuration commit to the code commit?